guardrails/validators.py

```diff
@@ -544,6 +545,19 @@ def validate(self, value: Any, metadata: Dict) -> ValidationResult:
             f"Validating {value} is in length range {self._min} - {self._max}..."
         )
 
+        # create a random string if the result is totally malformed
+        if not value or not (isinstance(value, str) or (isinstance(value, list))):
+            logger.debug(f"Value {value} is not a string or list")
```

Comment:
This part definitely belongs in schema verification.  By the time the llm response gets to our validators we should be assured that it is the correct type and shape.